config/types/root_types.ts

```diff
@@ -10,63 +14,78 @@ export type ValueOf<ObjectType> = ObjectType[keyof ObjectType];
 export type NullableString = string | null;
 
 export interface TopLevelDocument {
-  authors: Array<AuthorProfile>,
-  score: number,
-  createdDate: string,
-  discussionCount: number,
-  unifiedDocument: UnifiedDocument,
-  hubs: Array<Hub>,
-  createdBy: CreatedBy | null,
-  userVote?: "downvote" | "upvote" | "neutralvote" | null,
-  title?: string,
-  externalUrl?: string,
-  doi?: string,
-  datePublished?: string,
-  journal?: string,
-  formats?: Array<PaperFormat>,
-  note?: any,
-  markdown?: string,
-  isReady: boolean,
-  consensus?: CitationConsensus,
-  boostAmount: number,
-  id: ID,
+  authors: Array<AuthorProfile>;
+  score: number;
+  createdDate: string;
+  discussionCount: number;
+  unifiedDocument: UnifiedDocument;
+  hubs: Array<Hub>;
+  createdBy: CreatedBy | null;
+  userVote?: "downvote" | "upvote" | "neutralvote" | null;
```

Comment:
While at it, could you update this to use the new type you created? There may be a type issue that will happen due to the `null`